protocol/0095-HVMR-high_volume_maker_rebate.md

```diff
@@ -108,8 +112,20 @@ As the rebate possible level interacts with other fee settings there must be a r
+
+#### Capping the effective rebate factor
+
+1. If a party qualifies for a tier where the `additional_rebate_factor` is greater than (`buyback_fee_factor` + `treasury_fee_factor`), their `effective_additional_rebate_factor` should be capped to a maximum and set to (`buyback_fee_factor` + `treasury_fee_factor`). In this case no treasury or buyback fees should be collected by the network for trades involving this party. (<a name="0095-HVMR-029" href="#0095-HVMR-029">0095-HVMR-029</a>).
+
+1. If the `buyback_fee_factor` is increased through a proposal in the middle of an epoch such that a parties `additional_rebate_factor` is greater than (`buyback_fee_factor` + `treasury_fee_factor`), their `effective_additional_rebate_factor` should be set to the new maximum (`buyback_fee_factor` + `treasury_fee_factor`). (<a name="0095-HVMR-030" href="#0095-HVMR-030">0095-HVMR-030</a>).
+1. If the `treasury_fee_factor` is increased through a proposal in the middle of an epoch such that a parties `additional_rebate_factor` is greater than (`buyback_fee_factor` + `treasury_fee_factor`), their `effective_additional_rebate_factor` should be set to the new maximum (`buyback_fee_factor` + `treasury_fee_factor`). (<a name="0095-HVMR-031" href="#0095-HVMR-031">0095-HVMR-031</a>).
+1. If the `buyback_fee_factor` and `treasury_fee_factor` are both increased through a batch proposal in the middle of an epoch such that a parties `additional_rebate_factor` is greater than (`buyback_fee_factor` + `treasury_fee_factor`), their `effective_additional_rebate_factor` should be set to the new maximum (`buyback_fee_factor` + `treasury_fee_factor`). (<a name="0095-HVMR-032" href="#0095-HVMR-032">0095-HVMR-032</a>).
+
+1. If the `buyback_fee_factor` is reduced through a proposal in the middle of an epoch such that a parties `additional_rebate_factor` is less than (`buyback_fee_factor` + `treasury_fee_factor`), their `effective_additional_rebate_factor` should be set to their current `additional_rebate_factor`. (<a name="0095-HVMR-033" href="#0095-HVMR-033">0095-HVMR-033</a>).
+1. If the `treasury_fee_factor` is reduced through a proposal in the middle of an epoch such that a parties `additional_rebate_factor` is greater than (`buyback_fee_factor` + `treasury_fee_factor`), their `effective_additional_rebate_factor` should be set to their current `additional_rebate_factor`. (<a name="0095-HVMR-034" href="#0095-HVMR-034">0095-HVMR-034</a>).
+1. If the `buyback_fee_factor` and `treasury_fee_factor` are both reduced through a batch proposal in the middle of an epoch such that a parties current `additional_rebate_factor` is greater than (`buyback_fee_factor` + `treasury_fee_factor`), their `effective_additional_rebate_factor` should be set to their current `additional_rebate_factor`. (<a name="0095-HVMR-035" href="#0095-HVMR-035">0095-HVMR-035</a>).
```

Comment:
is `effective_additional_rebate_factor` not meant to be reset here?

Reply:
Ah thanks there is a typo in 033 and 034. Should say if rebate is less than the factors - fixing now.